doc/experiment.rst

```diff
@@ -1,327 +1,454 @@
+  initialize the Ensemble with a ``RunSettings`` object and
+  `params` argument. The run settings and parameters will
+  expand to all Ensemble members.
+
+Case 2 : Define the number of Models created.
+  If you would like to manually define the number of models created
+  within an Ensemble, initialize an ``Ensemble`` with a ``RunSettings`` object and
+  `replicas` argument. The `replicas` argument will create clones
+  of the ``Model`` object. The run settings will
+  expand to all Ensemble members.
+
+Case 3 : Launch as a batch job and manually append Models.
+  If you would like to launch the Ensemble as a batch job,
+  specify ``BatchSettings`` when initializing an ``Ensemble`` object.
+  An empty Ensemble object will be created and manually appending Models
+  is required.
+
+Case 4 : Launch as a batch job and test combinations/permutations.
+  If you would like to launch the Ensemble as a batch job and
+  define a set of parameter values
+  for which you want to test different combinations/permutations,
+  initialize an ``Ensemble`` with a ``BatchSettings``, ``RunSettings`` objects
+  and `params` argument. The run settings and parameters will
+  expand to all Ensemble members. The Ensemble will be launched as
+  a single batch job.
+
+Case 5 : Launch as a batch job and define number of Models created.
+  If you would like to launch the Ensemble as a batch job
+  manually define the number of models created
+  within an Ensemble, initialize an ``Ensemble`` with a ``RunSettings`` object and
+  `replicas` argument. The `replicas` argument will create clones
+  of the ``Model`` object. The run settings will
+  expand to all Ensemble members.
+
+The ``create_ensemble()`` factory method returns an initialized ``Ensemble`` object that
+gives you access to functions associated with the :ref:`Ensemble API<ensemble_api>`.
+
+.. note::
+  For more information on all cases and input arguments, navigate to the Ensemble documentation page.
+==================
+Experiment Example
+==================
+.. compound::
+  In the following subsections, we provide an example of using SmartSim to automate the
+  deployment of an HPC workload and distributed, in-memory storage, within
+  the workflow.
+
+  Continue to the example to:
+
+  .. list-table:: Experiment example contents
+   :widths: auto
+   :header-rows: 1
+
+   * - Initialize
+     - Start
+     - Stop
+   * - a workflow (``Experiment``)
+     - the in-memory database (``Orchestrator``)
+     - the in-memory database (``Orchestrator``)
+   * - a in-memory database (``Orchestrator``)
+     - the workload (``Model``)
+     - 
+   * - a workload (``Model``)
+     - 
+     - 
+
+Initialize
+----------
+.. compound::
+  To create a workflow, we *initialize* an ``Experiment`` object
+  once at the beginning of the Python driver script.
+  To create an Experiment, we specify a name
+  and the system launcher with which we will execute the driver script.
```

Comment:
"with which we will execute the driver script" -> "with which all entities will be executed."